### ISSUE TITLE
Discord issues

### ISSUE THREAD

**Issue by Luke-K0** [ClaraCrazy/Flipper-Xtreme#478]
Is there some problem with Discord channel? I tried to join several times on diferent devices and OSes - Linux (firefox, chrome, app), Windows (firefox, app),  Android(app), but no captcha shows when I'm trying to verify myself, just a popup with "ANSWER" textbox. Also "faq" and "support us" are not showing up when following steps.

**Comment by Willy-JL** [comment]
cant replicate. can you try again? do you have anything unusual about your network configuration that might block some websites?

**Comment by Luke-K0** [comment]
> cant replicate. can you try again? do you have anything unusual about your network configuration that might block some websites?

No, there are no blocks on my network configuration. I tried two different ISPs, mobile data, and vpn, with different mix of browsers/app. Everytime after i click "I'm an adult human beeing" I get 
"Please verify yourself to gain access to Flipper-Xtreme
Once you are ready to provide your answer click the button below.
NOTE: The captcha is CaSe SenSiTivE and does not include spaces." 
and when clicked "Answer" opens "Verify yourself" with info about sending form to Captcha.bot and textbox "ANSWER" "ABCDEF". No captcha loads, nothing to solve.

**Comment by ghost** [comment]
> No, there are no blocks on my network configuration. I tried two different ISPs, mobile data, and vpn, with different mix of browsers/app. Everytime after i click "I'm an adult human beeing" I get 
> 
> "Please verify yourself to gain access to Flipper-Xtreme
> 
> Once you are ready to provide your answer click the button below.
> 
> NOTE: The captcha is CaSe SenSiTivE and does not include spaces." 
> 
> and when clicked "Answer" it opens another dialog "Verify yourself" with info about sending form to Captcha.bot and textbox "ANSWER" "ABCDEF". No captcha loads, nothing to solve.

check your internet settings by going to 10.0.0.1 and logging in with user: "admin" and pass: "password" then change the password if you need to.

not sure if it works on all providers.

**Comment by theandrewlane** [comment]
I'm experiencing the same issue. Discord for iOS - on wifi or cellular 🤷🏻‍♂️

![image](https://github.com/Flipper-XFW/Xtreme-Firmware/assets/9087248/32776384-a106-4068-9f61-8d5ea9f03f7b)

**Comment by ClaraCrazy** [comment]
fix your client then lol. 30k ppl can use it fine, two cant

**Comment by CapuchaRojo** [comment]
I have same issue. Can’t access the discord from any device

Adam V. M.
971-757-4070

CEO @ A Storms Coming LLC
An Ai Automation Agency  https://www.astormscoming.com
Recruiter @ Valued Merchants
Find out more: https://hihello.me/p/a4903ad4-cc3c-4f0f-9826-1cdf6af0d204

**Comment by spd2last** [comment]
Same issue here... No Captcha made visible. Didn't see any errors, blocks, or anything obvious in the browser's console and I disabled all ads/dns filtering. Attempted in both Google Chrome browser and in Discord client.
<img width="619" alt="Screenshot 2024-02-28 at 5 31 55 PM" src="https://github.com/Flipper-XFW/Xtreme-Firmware/assets/101909381/af43cf7d-80ab-4991-94e2-e48b89f69220">

**Comment by domikazas** [comment]
same issue, i tried it on my pc on the we browser, still not working. I see this glitch has happened for many and its not our fault
